core/src/main/java/io/questdb/cairo/pool/AbstractMultiTenantPool.java

```diff
@@ -276,6 +198,102 @@ private void closeTenant(long thread, Entry<T> entry, int index, short ev, int r
         }
     }
 
+    private T get0(TableToken tableToken, @Nullable T copyOfTenant, boolean isCopyOf) {
```

Comment:
nit: `isCopyOf` should be same as `copyOfTenant != null` or is there any other case?

Reply:
Simplified in 669de9e2bc8ec868a31a702fdc6a32e3495a92de